Frameworks/include/UIColorInternal.h

```diff
@@ -17,13 +17,27 @@
 #pragma once
 
 #import <UIKit/UIColor.h>
-#import <CoreGraphics/CGPattern.h>
+#import <CoreGraphics/CoreGraphics.h>
+
+typedef struct {
+    CGFloat r;
+    CGFloat g;
+    CGFloat b;
+    CGFloat a;
+} ColorQuad;
```

Comment:
I had to back out the #658 merge because of broken test app.  This fixes all the issues.

---
In reply to: [71389054](https://github.com/Microsoft/WinObjC/pull/663#discussion_r71389054) [](ancestors = 71389054)